packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx

```diff
@@ -43,28 +43,27 @@ const Form = ({ connector }: Props) => {
             />
             <div className={styles.tip}>{t('connectors.guide.logo_tip')}</div>
           </FormField>
-          {!darkVisible && (
-            <Button
-              size="small"
-              type="text"
-              title="connectors.guide.logo_dark_show"
-              onClick={toggleDarkVisible}
-            />
-          )}
-          {darkVisible && (
-            <FormField title="connectors.guide.logo_dark">
-              <TextInput
-                placeholder={t('connectors.guide.logo_dark_placelholder')}
-                {...register('logoDark')}
+          {darkVisible ? (
+            <>
+              <FormField title="connectors.guide.logo_dark">
+                <TextInput
+                  placeholder={t('connectors.guide.logo_dark_placelholder')}
+                  {...register('logoDark')}
+                />
+                <div className={styles.tip}>{t('connectors.guide.logo_dark_tip')}</div>
+              </FormField>
+              <Button
+                size="small"
+                type="text"
+                title="connectors.guide.logo_dark_collapse"
+                onClick={toggleDarkVisible}
               />
-              <div className={styles.tip}>{t('connectors.guide.logo_dark_tip')}</div>
-            </FormField>
-          )}
-          {darkVisible && (
+            </>
+          ) : (
             <Button
               size="small"
               type="text"
-              title="connectors.guide.logo_dark_collapse"
+              title="connectors.guide.logo_dark_show"
               onClick={toggleDarkVisible}
```

Comment:
nit: These two buttons are almost identical, should just dynamically render the titles

Reply:
updated